client/blocks/inline-help/popover.jsx

```diff
@@ -88,10 +91,16 @@ class InlineHelpPopover extends Component {
 	};
 
 	render() {
-		const { translate } = this.props;
+		const { supportVariation, translate } = this.props;
 		const { showSecondaryView } = this.state;
 		const popoverClasses = { 'is-secondary-view-active': showSecondaryView };
 
+		const isForumSupportVariation = supportVariation === SUPPORT_FORUM;
+
+		const secondaryViewButtonLabel = isForumSupportVariation
+			? translate( 'Forums' )
```

Comment:
Note: I'm not sure what exactly the copy should be here so I've just left it as 'Forums', something indicative of the intent. If we decide we want to merge this change, we should think about improving on this label.